### ISSUE TITLE
Creator choices fast entry / form entry switching for columns in matrix question does not work with knockout 3.5.1

### ISSUE THREAD

**Issue by SamMousa** [surveyjs/survey-creator#677]
### Are you requesting a feature, reporting a bug or ask a question?
Bug

### What is the current behavior?
When adding a column to a matrix question switching between fast entry & form entry causes error.
I suspect this has to do with (rendering) changes in KnockoutJS)

### What is the expected behavior?
It should work.

### How would you reproduce the current behavior (if this is a bug)?
Create a survey, add a matrix question, go back and forth between fast & form entry.
Note that I cannot reproduce this when forcefully installing knockout 3.4.2.

Browser console output:
```
knockout-latest.js?v=v4.0.1:135 Uncaught TypeError: Cannot read property '$' of undefined
    at a.a.ec (knockout-latest.js?v=v4.0.1:135)
    at Object.G (knockout-latest.js?v=v4.0.1:41)
    at x (knockout-latest.js?v=v4.0.1:127)
    at a.ic.lc (knockout-latest.js?v=v4.0.1:129)
    at Function.notifySubscribers (knockout-latest.js?v=v4.0.1:38)
    at a.ic.c [as lc] (knockout-latest.js?v=v4.0.1:48)
    at Function.notifySubscribers (knockout-latest.js?v=v4.0.1:38)
    at Function.xa (knockout-latest.js?v=v4.0.1:43)
    at t.c [as koVisibleColumns] (knockout-latest.js?v=v4.0.1:42)
    at t.onColumnsChanged (survey.ko.min.js?v=v4.0.1:6)
a.a.ec @ knockout-latest.js?v=v4.0.1:135
G @ knockout-latest.js?v=v4.0.1:41
x @ knockout-latest.js?v=v4.0.1:127
(anonymous) @ knockout-latest.js?v=v4.0.1:129
notifySubscribers @ knockout-latest.js?v=v4.0.1:38
c @ knockout-latest.js?v=v4.0.1:48
notifySubscribers @ knockout-latest.js?v=v4.0.1:38
xa @ knockout-latest.js?v=v4.0.1:43
c @ knockout-latest.js?v=v4.0.1:42
t.onColumnsChanged @ survey.ko.min.js?v=v4.0.1:6
(anonymous) @ survey.ko.min.js?v=v4.0.1:6
e.propertyValueChanged @ survey.ko.min.js?v=v4.0.1:6
i.splice @ survey.ko.min.js?v=v4.0.1:6
t.updateArrayValue @ survey-creator.min.js?v=v4.0.1:12
t.updateItems @ survey-creator.min.js?v=v4.0.1:12
t.updateItemsTextOnTyping @ survey-creator.min.js?v=v4.0.1:12
(anonymous) @ survey-creator.min.js?v=v4.0.1:12
notifySubscribers @ knockout-latest.js?v=v4.0.1:38
(anonymous) @ knockout-latest.js?v=v4.0.1:39
setTimeout (async)
setTimeout @ knockout-latest.js?v=v4.0.1:16
(anonymous) @ knockout-latest.js?v=v4.0.1:8
c.qc @ knockout-latest.js?v=v4.0.1:39
Ia @ knockout-latest.js?v=v4.0.1:56
Hc @ knockout-latest.js?v=v4.0.1:54
notifySubscribers @ knockout-latest.js?v=v4.0.1:38
xa @ knockout-latest.js?v=v4.0.1:43
c @ knockout-latest.js?v=v4.0.1:42
t.updateItemsText @ survey-creator.min.js?v=v4.0.1:12
(anonymous) @ survey-creator.min.js?v=v4.0.1:12
notifySubscribers @ knockout-latest.js?v=v4.0.1:38
xa @ knockout-latest.js?v=v4.0.1:43
c @ knockout-latest.js?v=v4.0.1:42
o.changeToTextViewClick @ survey-creator.min.js?v=v4.0.1:12
(anonymous) @ knockout-latest.js?v=v4.0.1:100
build?id=2923:1 [DOM] Found 2 elements with non-unique id #sq_126i_Row 1_2: (More info: https://goo.gl/9p2vKq) <input type=​"radio" data-bind=​"css:​ question.koCss()​.itemValue, attr:​ {name:​ row.fullName, 'aria-required':​ question.isRequired, 'aria-label':​ question.locTitle.renderedHtml, value:​ '' + $data.value, id:​ question.inputId + '_' + row.name + '_' + $index()​}​, checked:​ row.koValue, enable:​ !question.isReadOnly" name=​"sq_126_Row_1" aria-label=​"question1" value=​"3" id=​"sq_126i_Row 1_2">​ <input type=​"radio" data-bind=​"css:​ question.koCss()​.itemValue, attr:​ {name:​ row.fullName, 'aria-required':​ question.isRequired, 'aria-label':​ question.locTitle.renderedHtml, value:​ '' + $data.value, id:​ question.inputId + '_' + row.name + '_' + $index()​}​, checked:​ row.koValue, enable:​ !question.isReadOnly" name=​"sq_126_Row_1" aria-label=​"question1" value=​"3" id=​"sq_126i_Row 1_2">​
build?id=2923:1 [DOM] Found 2 elements with non-unique id #sq_126i_Row 2_2: (More info: https://goo.gl/9p2vKq) <input type=​"radio" data-bind=​"css:​ question.koCss()​.itemValue, attr:​ {name:​ row.fullName, 'aria-required':​ question.isRequired, 'aria-label':​ question.locTitle.renderedHtml, value:​ '' + $data.value, id:​ question.inputId + '_' + row.name + '_' + $index()​}​, checked:​ row.koValue, enable:​ !question.isReadOnly" name=​"sq_126_Row_2" aria-label=​"question1" value=​"3" id=​"sq_126i_Row 2_2">​ <input type=​"radio" data-bind=​"css:​ question.koCss()​.itemValue, attr:​ {name:​ row.fullName, 'aria-required':​ question.isRequired, 'aria-label':​ question.locTitle.renderedHtml, value:​ '' + $data.value, id:​ question.inputId + '_' + row.name + '_' + $index()​}​, checked:​ row.koValue, enable:​ !question.isReadOnly" name=​"sq_126_Row_2" aria-label=​"question1" value=​"3" id=​"sq_126i_Row 2_2">​
```

### Specify your

* browser: chrome latest
* editor version: v1.5.10
* knockout verision: 3.5.1

**Comment by SamMousa** [comment]
Related to #676

**Comment by tsv2013** [comment]
I've reproduced the issue. Here is the plunker https://next.plnkr.co/edit/ztL3N3X6kKU1oIeD

**Comment by tsv2013** [comment]
Need to be fixed in https://github.com/surveyjs/survey-library as you can see from the commit.

Fix will be available in the nearest update, 1.5.11
